jujugui/static/gui/src/app/components/machine-view/test-machine-view.js

```diff
@@ -64,6 +64,7 @@ describe('MachineView', function() {
         createMachine={sinon.stub()}
         destroyMachines={sinon.stub()}
         environmentName="My Env"
+        jujuCoreVersion="1.4"
```

Comment:
We're moving towards setting everything as Juju 2 for the default case. Sorry that's going to make these test changes a little more tedious but this should instead be `jujuCoreVersion="2.0"`. And the rest below.